h5py/tests/test_file2.py

```diff
@@ -273,3 +275,35 @@ def test_no_track_order(self):
         self.populate(f)
         self.assertEqual(list(f),
                          sorted([str(i) for i in range(100)]))
+
+
+class TestFileMetaBlockSize(TestCase):
+
+    """
+        Feature: The minimum metadata block size can be set.
+    """
+
+    def test_file_create_with_meta_block_size(self):
+        meta_block_size = 4096
+        with File(
+            self.mktemp(), 'w',
+            meta_block_size=meta_block_size,
+            libver="latest"
+        ) as f:
+            f["test"] = 5
+            self.assertEqual(f.meta_block_size, meta_block_size)
+            self.assertGreaterEqual(f["test"].id.get_offset(), meta_block_size)
+
+    @pytest.mark.skipif(h5py.version.hdf5_version_tuple < (1, 10, 2),
+                        reason="HDF5 header became smaller in version v1.8")
+    def test_file_create_with_meta_block_size_libver(self):
+        meta_block_size = 512
+        libver = "v108"
```

Comment:
This option exists to manipulate how HDF5 allocates space in the file. That's the point.

Reply:
The default `meta_block_size` is 2048 bytes 

Thus we need two tests here:
1) One greater than the default `meta_block_size`: 4096 bytes
2) One less than the default `meta_block_size`: 512 bytes

I have removed version restrictions on both tests. They both now use `assertGreaterEqual` so that both tests will pass on HDF5 1.8. I added an upper bound on the 512 byte test to differentiate it from the situation when the default `meta_block_size` is used.